mod/b_extended_profile/views/default/profile/owner_block.php

```diff
@@ -32,7 +32,7 @@
 // @todo: create a link to edit the user profile picture
 
 // just dont display anything with icons when gsa hits page
-if (strstr(strtolower($_SERVER['HTTP_USER_AGENT']), 'gsa-crawler') === false) {
+if (strstr(strtolower($_SERVER['HTTP_USER_AGENT']), 'gsa-crawler') === false || strstr(strtolower($_SERVER['HTTP_USER_AGENT']), 'solr-crawler') === false) {
```

Comment:
Wouldn't this always evaluate to true unless $_SERVER['HTTP_USER_AGENT'] is both a gsa AND a solr crawler?

Reply:
Thanks.. Updated 👍